client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -34,33 +33,25 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( { dispute } ) => {
 		<div className="transaction-details-dispute-details-wrapper">
 			<Card>
 				<CardBody className="transaction-details-dispute-details-body">
-					{ isAwaitingResponse( dispute.status ) &&
```

Comment:
Note: `isAwaitingResponse` is handled in the parent component:

https://github.com/Automattic/woocommerce-payments/blob/050ee037ba7575f04477d6451ee16400823720ef/client/payment-details/summary/index.tsx#L381-L391